### ISSUE TITLE
autodoc_default_options should accept True values

### ISSUE THREAD

**Issue by jwodder** [sphinx-doc/sphinx#5459]
**Short version**: Instead of this:

```
autodoc_default_options = {"members": None}
```

it would make more sense to be able to write this:

```
autodoc_default_options = {"members": True}
```

---

The new `autodoc_default_options` config option currently requires that its `dict` values be either strings or `None`, where `None` "means 'yes/true/on'".  Seeing as `None` means `True`, it would make more sense to be able to actually write `True` instead (but keep `None` around for backwards compatibility).

Currently (in Sphinx 1.8), setting an `autodoc_default_options` value to `True` just produces an unhelpful error message:
```
Exception occurred:
  File "/Users/jwodder/work/GITHUB/headerparser/.tox/docs/lib/python3.7/site-packages/sphinx/ext/autodoc/__init__.py", line 82, in members_option
    return [x.strip() for x in arg.split(',')]
AttributeError: 'bool' object has no attribute 'split'
The full traceback has been saved in /var/folders/l7/wrkq93d133d8zpn36fmqrq0r0000gn/T/sphinx-err-dld5die4.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

**Comment by tk0miya** [comment]
Fixed by #6053.
Thank you for your proposal!

**Comment by Borda** [comment]
When it will be in a release?

**Comment by tk0miya** [comment]
It was already released as v2.0.0 in last April.
